### PR TITLE
[BugFix] refactor the error message when mv is inactivated

### PR DESCRIPTION
**Pull request by murphyatwork**
## Why I'm doing:
- the error message when mv gets inactivated is not clear

## What I'm doing:

Fixes #issue

## What type of PR is this:

- [x] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [x] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

**Comment by mofeiatwork** [comment]
@Mergify rebase

**Comment by mofeiatwork** [comment]
@Mergify rebase

File: fe/fe-core/src/main/java/com/starrocks/common/MaterializedViewExceptions.java [hunk 1]
```diff
@@ -45,7 +52,23 @@ public static String inactiveReasonForBaseViewChanged(String tableName) {
         return "base-view changed: " + tableName;
     }
 
+    public static String inactiveReasonForBaseInfoMissed() {
+        return "base-info missed";
+    }
+
+    public static String inactiveReasonForDbNotExists(long dbId) {
```
Comment (allencelerdata) [inline fe/fe-core/src/main/java/com/starrocks/common/MaterializedViewExceptions.java:59]: It's better to show the database name in the error message.

**Review by dirtysalt — APPROVED** [review]

File: fe/fe-core/src/main/java/com/starrocks/common/MaterializedViewExceptions.java (hunk 1 above)
Comment (chaoyli) [inline fe/fe-core/src/main/java/com/starrocks/common/MaterializedViewExceptions.java:59]:
```
It's better to show the database name in the error message.
```

**Review by kangkaisen — APPROVED** [review]

**Review by LiShuMing — APPROVED** [review]

**Review by nshangyiming — APPROVED** [review]

**Pull request merged**